code/modules/urist/random.dm

```diff
@@ -5,6 +5,22 @@
 		return 0
 	return 1
 
+//vg color setup stuff
+
+var/default_colour_matrix = list(1,0,0,0,\
+								 0,1,0,0,\
+								 0,0,1,0,\
+								 0,0,0,1)
+
+var/global/list/NOIRMATRIX = list(0.66,0.33,0.33,0,\
```

Comment:
As per usual Urist isn't exactly following Bay's code standards so I wouldn't worry to much, however, if you'd like you can make this into a `GLOBAL_LIST_INIT(noirmatrix, list(...))` You'll have to refer to it as GLOB.noirmatrix if you do.

Reply:
Yeah, I've been making all new global lists into those, so I'd appreciate if it was done here as well. It's a little nicer when referring to them elsewhere.